src/main/java/org/jitsi/meet/ComponentMain.java

```diff
@@ -182,12 +184,28 @@ private ClassLoader loadBundlesJars(OSGiBundleConfig bundleConfig) {
         String bundlesJarsPath = bundleConfig.getBundlesJarsPath();
         if (bundlesJarsPath == null)
         {
-            return ClassLoader.getSystemClassLoader();
```

Comment:
@sidorovis Why do we need this?

I was testing running our component on java 11 and this change using the class loader returned by getPlatformClassLoader breaks them.

Reply:
I'm currently testing and plan to remove it.

Reply:
I see in the comments of https://github.com/jitsi/jitsi-android-osgi/pull/1 it was a discussion with @ibauersachs 
In jitsi-android-osgi we want the classloader that has access to the whole classpath, which is not the case for Felix.  Our lib is just a wrapper so we can reuse and run the OSGi bundles in a normal classLoader way ... 
I just tested and everything works with just ClassLoader.getSystemClassLoader() when using java11. There are few more minor modifications and I will be creating a PR soon.